geomstats/geometry/grassmannian.py

```diff
@@ -50,9 +50,33 @@ def belongs(self, point, tolerance=TOLERANCE):
         -------
         belongs : bool
         """
-        raise NotImplementedError(
-            'The Grassmann `belongs` is not implemented.'
-            'It shall test whether p*=p, p^2 = p and rank(p) = k.')
+        point = gs.to_ndarray(point, to_ndim=3)
```

Comment:
Sorry that you have to do this reshaping. We are currently working on refactorizing the codebase to get rid of all of these.